pkgs/by-name/go/go-graft/package.nix

```diff
@@ -8,20 +13,36 @@ buildGoModule rec {
     owner = "mzz2017";
     repo = "gg";
     rev = "v${version}";
-    sha256 = "sha256-DXW0NtFYvcCX4CgMs5/5HPaO9f9eFtw401wmJdCbHPU=";
+    hash = "sha256-DXW0NtFYvcCX4CgMs5/5HPaO9f9eFtw401wmJdCbHPU=";
   };
 
   env.CGO_ENABLED = 0;
 
-  ldflags = [ "-X github.com/mzz2017/gg/cmd.Version=${version}" "-s" "-w" ];
+  ldflags = [
+    "-X github.com/mzz2017/gg/cmd.Version=${version}"
+    "-s"
+    "-w"
+  ];
+
   vendorHash = "sha256-fnM4ycqDyruCdCA1Cr4Ki48xeQiTG4l5dLVuAafEm14=";
-  subPackages = [ "." ];
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  postInstall = ''
+    installShellCompletion --cmd gg \
+    --bash completion/bash/gg \
+    --fish completion/fish/gg.fish \
+    --zsh completion/zsh/_gg
+  '';
 
   meta = with lib; {
     description = "Command-line tool for one-click proxy in your research and development without installing v2ray or anything else";
     homepage = "https://github.com/mzz2017/gg";
     license = licenses.agpl3Plus;
```

Comment:
I have some questions about the license filled here. The repository's license file indicates that it seems to be [agpl3Only](https://github.com/mzz2017/gg?tab=AGPL-3.0-1-ov-file#readme). CC @XYenon

Reply:
Oh, that should be my mistake. Just change it to agpl3Only.